backend/tests/helpers/query_benchmark/data_generator.py

```diff
@@ -1,13 +1,13 @@
 from abc import abstractmethod
 from pathlib import Path
-from typing import Callable
+from typing import Callable, Optional
```

Comment:
Just a quick note here, we don't have any type of controls in place yet but we've typically moved away from the `Optional` import for new code within infrahub-server I.e. `Optional[str]` gets replaced by `str  | None`. This is more FYI and not something that you strictly have to change, in some point in the future we'll probably clear this out with a linter. Same thing with `from typing import List`.